samples/KeyVault/disallow-keyvault-access-policies/README.md

```diff
@@ -0,0 +1,25 @@
+# Disallow Key Vault data plane access policies
+
+Audits Key Vault vaults if their data plane access policies grant access to a specified AAD object. More information on Key Vault access management is available [here](https://docs.microsoft.com/en-us/azure/key-vault/key-vault-secure-your-key-vault).
+
+## Try on Portal
+
+[![Deploy to Azure](http://azuredeploy.net/deploybutton.png)](https://portal.azure.com/#blade/Microsoft_Azure_Policy/CreatePolicyDefinitionBlade/uri/https%3A%2F%2Fraw.githubusercontent.com%2FAzure%2Fazure-policy%2Fmaster%2Fsamples%2FKeyVault%2Fdisallow-keyvault-access-policies%2Fazurepolicy.json)
+[![Deploy to Azure Gov](https://docs.microsoft.com/azure/governance/policy/media/deploy/deployGovbutton.png)](https://portal.azure.us/?#blade/Microsoft_Azure_Policy/CreatePolicyDefinitionBlade/uri/https%3A%2F%2Fraw.githubusercontent.com%2FAzure%2Fazure-policy%2Fmaster%2Fsamples%2FKeyVault%2Fdisallow-keyvault-access-policies%2Fazurepolicy.json)
+
+## Try with PowerShell
+
+````powershell
+$definition = New-AzPolicyDefinition -Name "disallow-keyvault-access-policies" -DisplayName "Audit when a given service principal is assigned to the Key Vault data plane" -description "Audit when a specified AAD object is granted any permissions to secrets, certs, or keys stored in a Key Vault through data plane access policies." -Policy 'https://raw.githubusercontent.com/Azure/azure-policy/master/samples/KeyVault/disallow-keyvault-access-policies/azurepolicy.rules.json' -Mode All
+$definition
+$assignment = New-AzPolicyAssignment -Name <assignmentname> -Scope <scope> -PolicyDefinition $definition
+$assignment 
```

Comment:
assignment examples should include parameters (i.e. set a value for aadObjectId)

Reply:
See [copy-resource-group-tag](https://github.com/Azure/azure-policy/tree/master/samples/ResourceGroup/copy-resourcegroup-tag) for a good example

---
In reply to: [294006735](https://github.com/Azure/azure-policy/pull/337#discussion_r294006735) [](ancestors = 294006735)